resources/building-mobile-extensions/event-listeners.md

```diff
@@ -238,13 +295,26 @@ In iOS the event listeners are registered using the `registerListener` method of
 @end
 ```
 
-### **Android**
+{% endtab %}
+{% endtabs %}
 
-Event listeners in Android are registered using the `registerEventListener` method of the `ExtensionApi` interface. You can access this interface by using the `getApi` method in `Extension`. This example show registering the listener from your extension contructor.
+## Registering a Wildcard Listener
+
+If you need to listen for all the events received and broadcasted by the Event Hub, you can register a wildcard listener by using the `registerWildcardListener` API.
```

Comment:
Please change the beginning of this sentence to read "To listen for all events that are received and broadcasted..."